docs/build/html/_sources/methods.rst.txt

```diff
@@ -291,42 +291,56 @@ All other outputs are ``time series x XYZ`` where XYZ is the serialized list of
 with two conductances each: ``NaV`` and ``Kd``, the intrinsic currents would return in the form of a ``time steps x 4`` matrix where the columns would be ordered:
 Kd current from AB, NaV current from AB, Kd current from BC, NaV current from BC.
 
-If ``integrate`` is passed one argument, it interprets this as a matrix of injected currents (in nA). For example ::
+If the ``xolotl`` properties ``I_ext`` and ``V_clamp`` are empty (i.e. ``x.I_ext = []``), the integration proceeds without injected current or clamped voltage.
+Injected current and voltage clamp cannot co-occur.  Setting one of these properties negates the other. If ``I_ext`` is a scalar,
+current is injected into each compartment at every time step. ::
 
-  % applies 0.2 nA to each compartment
-  x.integrate(0.2)
-  % applies 0.2 nA to first compartment of three
-  x.integrate([0.2 0 0])
-  % applies a time-varying current to first compartment
-  % where S is a matrix of size time steps × compartments
-  x.integrate(S)
+  % add 0.1 nA to each compartment at each time-step
+  x.I_ext = 0.1;
 
-If ``integrate`` is passed two arguments, the first is ignored, and the second is treated as a matrix of clamped voltage. ::
+If ``I_ext`` is non-scalar, it must be a matrix of dimension ``nSteps x nComps`` where
```

Comment:
again, not precisely true. you can give it a Nx1 vector

Reply:
done